assets/src/design-system/icons/dashboard/index.js

```diff
@@ -18,8 +18,16 @@
  * Internal dependencies
  */
 
+export {
+  CompassActive,
+  FloppyDiskActive,
+  HomeWithHeartActive,
+} from './activeIcons';
 export { ReactComponent as ExploreTemplatesActive } from './exploreTemplatesActive.svg';
 export { ReactComponent as ExploreTemplatesNormal } from './exploreTemplatesNormal.svg';
+export { ReactComponent as Compass } from './compass.svg';
+export { ReactComponent as FloppyDisk } from './floppy_disk.svg';
+export { ReactComponent as HomeWithHeart } from './home_with_heart.svg';
 export { ReactComponent as SavedTemplatesActive } from './savedTemplatesActive.svg';
 export { ReactComponent as SavedTemplatesNormal } from './savedTemplatesNormal.svg';
 export { ReactComponent as StoriesActive } from './storiesActive.svg';
```

Comment:
Since you replaced the need for duplicate SVGs here, can you remove anything in icons/dashboard that is not used for the nav? These are only used there so we don't need the dupes.